src/Models/Market.php

```diff
@@ -57,42 +58,42 @@ public function locations()
 
     public function competitors()
     {
-        return $this->hasMany(Competitor::class);
+        return $this->hasMany(config('locations.model_class.competitor'));
     }
 
     public function rooms()
     {
-        return $this->hasManyThrough(Room::class, Location::class);
+        return $this->hasManyThrough(config('locations.model_class.room'), Location::class);
```

Comment:
@drewroberts so we add hasManyThrough to support BaseModel?

Reply:
Yes. Has this been added?